packages/vue/src/components/molecules/SfCollectedProduct/SfCollectedProduct.html

```diff
@@ -0,0 +1,40 @@
+<div class="sf-collected-product">
+  <!-- @slot -->
+  <slot name="remove">
+    <div class="sf-collected-product__remove" @click="removeProduct">
```

Comment:
We have a component exactly for this use case (SfCircleIcon). Please use this one for mobile and just icon on desktop

Reply:
ok i will use it